components/component/team-members-page-view.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { useState } from 'react';
+import { Button, Checkbox, Modal, Table } from 'flowbite-react';
```

Comment:
I think next-ui covered all these components. If it is possible, please replace from the next-ui

Reply:
Replaced these to the next-ui components